shard/committee/assignment.go

```diff
@@ -123,6 +142,7 @@ func NewEPoSRound(stakedReader StakingCandidatesReader) (
 
 func prepareOrders(
 	stakedReader StakingCandidatesReader,
+	snapshotOnly bool,
```

Comment:
no bool parameters please. Just make two functions for the branch. prepareOrdersSnapshot, prepareOrders